internal/cmd/common.go

```diff
@@ -276,3 +279,80 @@ type runFunc func(context.Context) error
 const tlsFileMode = os.FileMode(int(0o700))
 
 var defaultTLSDir = filepath.Join(GetDefaultConfigHome(), "tls")
+
+func promptEnvVars(cmd *cobra.Command, runBlocks []project.FileCodeBlock) error {
```

Comment:
```suggestion
func promptEnvVars(cmd *cobra.Command, runBlocks... project.FileCodeBlock) error {
```

This syntax prevents the awkward conversion to a list for single blocks